### PR TITLE
Product category (version with the corrections)

### PR DESCRIPTION
**Pull request by ue71603**
Messed up with desktop git, so new pull request

### CODE REVIEW

**Comment by pvgrumbkow** [comment]
Am I right, and your pull request contains only the definition of ProductCategoryStructure and not its usage?

**Comment by ue71603** [comment]
You are right, but it was not intended. Updated the rest now.

**Comment by pvgrumbkow** [comment]
Sorry, now you screwed OJP_commons.xsd completely up.

**Comment by ue71603** [comment]
you are right. But how? I edited it directly in github. It looks even ok in the comparison
https://github.com/VDVde/OJP/pull/95/commits/ce6e5dc7be1936b9510f6b6380f725d3a506cea0

**Comment by pvgrumbkow** [comment]
It's still there. You removed OJP_commons.xsd and added an OJP_commons.xsd.tmp.

**Comment by ue71603** [comment]
I used the github inline editor, which obviously doesn't do what it should.  And we currently seem to try to outcommit each other 👍

**Comment by ue71603** [comment]
Now, it looks good to me.

File: OJP_JourneySupport.xsd [hunk 1]
```diff
@@ -58,6 +80,11 @@
 					<xs:documentation>[a specialisation of MODE in TMv6] an extended range of VEHICLE MODEs, aggregating them with some SUBMODEs</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="ProductCategory" type="ProductCategoryStructure">
+				<xs:annotation>
+					<xs:documentation>A product category for the service.</xs:documentation>
```
Comment (sgrossberndt) [inline OJP_JourneySupport.xsd:85]: Add an example please

**Comment by ue71603** [comment]
In general ok
* Add a reference to NeTEx / Siri
* Explain what a difference what a product category is.
* With examples 
* More descriptive text 
Then update pull request, approved when enough positive feedback.

File: OJP_JourneySupport.xsd [hunk 2]
```diff
@@ -23,22 +23,22 @@
...
 			<xs:element name="Name" type="InternationalTextStructure" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Name of the product category.</xs:documentation>
+					<xs:documentation>Name of the product category. This is the full name for this product category. E.g. "Autoreisezug" in Switzerland or "Dampfschiff"</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="ShortName" type="InternationalTextStructure" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Short name or acronym of product category.</xs:documentation>
+					<xs:documentation>Short name or acronym of product category. If you intend to publish the product category, this is were you would put, the information that you want to associate with the journey. E.g. "ARZ" in Switzerland or "BAV".</xs:documentation>
```
Comment (sgrossberndt) [inline OJP_JourneySupport.xsd:36]: What is meant by `this is were you would put, the information that you want to associate with the journey`?

File: OJP_JourneySupport.xsd [hunk 3]
```diff
@@ -23,22 +23,22 @@
...
 				</xs:annotation>
 			</xs:element>
 			<xs:element ref="ProductCategoryRef" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>A reference to the product category.</xs:documentation>
+					<xs:documentation>A reference to the product category. This is the internal code that is used within the reference frameworks (NeTEx, Siri). It is usually not displayed, but interpreted by the technical system. E.g. ch:1:TypeOfProductCategory:ARZ (Autoreisezug in Switzerland) or ch:1:TypeOfProductCategory:BAV. </xs:documentation>
```
Comment (sgrossberndt) [inline OJP_JourneySupport.xsd:41]: `This is the internal code used within the reference frameworks (NeTEx, SIRI)`

File: OJP_JourneySupport.xsd [hunk 4]
```diff
@@ -82,7 +82,7 @@
 			</xs:element>
 			<xs:element name="ProductCategory" type="ProductCategoryStructure">
 				<xs:annotation>
-					<xs:documentation>A product category for the service.</xs:documentation>
+					<xs:documentation>A product category for the service. This is a classification that exists in NeTEx/Siri that can be used to identify groups of journeys with some special properties for marketing and fare products. E.g. "TE2" for SNCF or a special panorama train "PE" in Switzerland. </xs:documentation>
```
Comment (sgrossberndt) [inline OJP_JourneySupport.xsd:85]: This is a classification defined in NeTEx/SIRI used to identify groups of journeys with some special properties for marketing and fare products, e.g. "TE2" for SNCF or a special panorama train "PE" in Switzerland.

**Review by sgrossberndt — CHANGES_REQUESTED** [review]

File: OJP_Common.xsd [hunk 5]
```diff
@@ -95,7 +95,7 @@
 	</xs:simpleType>
 	<xs:element name="ProductCategoryRef" type="siri:ProductCategoryRefStructure">
 		<xs:annotation>
-			<xs:documentation>Reference to a product category.</xs:documentation>
+			<xs:documentation>Reference to a product category. It is useful to use the same types that are also used in time tables and real time data.</xs:documentation>
```
[the remark this answers is not in the document]
Comment (ue71603) [reply OJP_Common.xsd:98]: Changed the text to 
Reference to a product category. The product categories should be defined and used in a general manner over all channels (e.g. NeTEx, Siri, OJP)

File: OJP_JourneySupport.xsd [hunk 6]
```diff
@@ -23,22 +23,22 @@
...
 	<xs:complexType name="ProductCategoryStructure">
 		<xs:annotation>
-			<xs:documentation>Product category based on NeTEx</xs:documentation>
+			<xs:documentation>Product category based on NeTEx. A product category is a classification to particular VEHICLE JOURNEYs to express some common properties of journeys form marketing and fare products. </xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
 			<xs:element name="Name" type="InternationalTextStructure" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Name of the product category.</xs:documentation>
+					<xs:documentation>Name of the product category. This is the full name for this product category. E.g. "Autoreisezug" in Switzerland or "Dampfschiff"</xs:documentation>
```
[the remark this answers is not in the document]
Comment (ue71603) [reply OJP_JourneySupport.xsd:31]: changed

File: OJP_JourneySupport.xsd [hunk 7]
```diff
@@ -23,22 +23,22 @@
 	</xs:complexType>
 	<xs:complexType name="ProductCategoryStructure">
 		<xs:annotation>
-			<xs:documentation>Product category based on NeTEx</xs:documentation>
+			<xs:documentation>Product category based on NeTEx. A product category is a classification to particular VEHICLE JOURNEYs to express some common properties of journeys form marketing and fare products. </xs:documentation>
```
[the remark this answers is not in the document]
Comment (ue71603) [reply OJP_JourneySupport.xsd:26]: changed

File: OJP_Common.xsd [hunk 8]
```diff
@@ -81,7 +81,7 @@
 	</xs:annotation>
 	<xs:complexType name="ProductCategoryRefStructure">
 		<xs:annotation>
-			<xs:documentation>Reference to a product category.</xs:documentation>
+			<xs:documentation>Reference to a product category.ProductCateogry is defined in NeTEx (and also in Siri) as a classification for VEHICLE JOURNEYs to express some common properties of journeys fo rmarketing and fare products. E.g. to identify boats that run with steam in Switzerland ("BAV")</xs:documentation>
```
[the remark this answers is not in the document]
Comment (ue71603) [reply OJP_Common.xsd:84]: changed

File: OJP_JourneySupport.xsd (hunk 2 above)
Comment (ue71603) [reply OJP_JourneySupport.xsd:36]: changed to: hort name or acronym of product category. The short name  will likely be published. E.g. the product category is more important for publication in Switzerland than Mode / Submode. Examples "BAV", "ARZ", "TGV"

File: OJP_JourneySupport.xsd (hunk 3 above)
Comment (ue71603) [reply OJP_JourneySupport.xsd:41]: changed.

File: OJP_JourneySupport.xsd (hunk 4 above)
Comment (ue71603) [reply OJP_JourneySupport.xsd:85]: changed

**Review by sgrossberndt — APPROVED** [review]

**Review by pvgrumbkow — APPROVED** [review]

**Pull request merged**